mods/post_data.js

```diff
@@ -98,7 +98,16 @@ PostDataBucket.prototype.putRevision = function(restbase, req) {
             throw e;
         }
     });
+};
 
+PostDataBucket.prototype.calculateHash = function(restbase, req) {
```

Comment:
I almost wonder if we should hook this up as `$.hash()`, saving the extra endpoint.